modules/osdk-scorecard-store-output.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-scorecard.adoc
+
+:osdk_ver: v1.16.0
+
+:_content-type: MODULE
+[id="osdk-scorecard-store-output_{context}"]
+= Local storage of scorecard results
+
+You can enable local storage of scorecard test results by using the `scorecard` command or by modifying your `config.yaml` file.
```

Comment:
This sounds like the abstract for a procedure module. I strongly suggest refactoring the content into discrete steps that follow the procedure template.